3-reveal/migrations/4-FI/1-Thailand-2019/deploy/events_jurisdictions_materialized_view.psql

```diff
@@ -0,0 +1,37 @@
+-- Deploy reveal_fi_thailand_2019:events_jurisdictions_materialized_view to pg
+-- requires: reveal_transaction_tables:events
+-- requires: reveal_transaction_tables:plans
+-- requires: reveal_transaction_tables:goals
+-- requires: reveal_transaction_tables:plan_jurisdiction
+-- requires: reveal_transaction_tables:jurisdictions
+-- requires: utils:refresh_mat_views
+
+BEGIN;
+
+SET search_path TO :"schema",public;
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS events_jurisdictions_materialized_view
+AS
+SELECT
+    events.id,
+    events.details->'case_classification' AS case_classification,
+    events.details->'case_number' AS case_number,
+    events.details->'investigtion_date' AS date_of_diagnosis,
+    plans.identifier AS plan_id,
+    goals.goal_id,
+    plan_jurisdiction.jurisdiction_id,
+    jurisdictions.name AS jurisdiction_name,
+    jurisdictions.parent_id AS jurisdiction_parent_id
+FROM events
+LEFT JOIN plans
+    ON plans.fi_opensrpeventid = events.id
+LEFT JOIN goals
+    ON goals.plan_id = plans.identifier
+LEFT JOIN plan_jurisdiction
+    ON plan_jurisdiction.plan_id = plans.identifier
```

Comment:
We might not need this join.  The location_id field on the events table should point to the jurisdiction_id that we want.  Please verify and if this is the case, change this.